src/Chapter2.hs

```diff
@@ -621,8 +640,10 @@ Write a function that takes elements of a list only in even positions.
 >>> takeEven [2, 1, 3, 5, 4]
 [2,3,4]
 -}
-takeEven = error "takeEven: Not implemented!"
-
+takeEven :: [a] -> [a]
+takeEven [] = []
+takeEven (x:xs:xss) = x : takeEven xss
+takeEven (x:xs) = [x] 
```

Comment:
In here `xs` is only could be `[]` as all other cases are covered above 🙂 So you can right it so:
```suggestion
takeEven [x] = [x] 
```